### PR TITLE
Update CDN link to the widget

### PR DESCRIPTION
**Pull request by Zmoki**

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ History
 x.x.x
 ~~~~~
 
-- Updated widget to version 2.10.2 (see `widget changelog`_).
+- Update CDN link to the widget and set wildcard version
```
Comment (dmitry-mukhin) [inline HISTORY.rst]: I'd like you to mention that this is a latest "2" version, not anything else.

**Comment by Zmoki** [comment]
@dmitry-mukhin bump

File: HISTORY.rst [hunk 2]
```diff
@@ -6,7 +6,8 @@ History
 x.x.x
 ~~~~~
 
-- Updated widget to version 2.10.2 (see `widget changelog`_).
+- Update CDN link to the widget
+- Use wildcard for get latest `2.x.x` version of widget
```
Comment (dmitry-mukhin) [inline HISTORY.rst]: ссылку на ченжлог можно оставить
"2.x will always get the latest widget version 2, read changelog now and then :)"

**Comment by Zmoki** [comment]
@dmitry-mukhin bump

File: pyuploadcare/dj/conf.py [hunk 3]
```diff
@@ -24,15 +24,15 @@
     conf.cdn_base = settings.UPLOADCARE['cdn_base']
 
 
-widget_version = settings.UPLOADCARE.get('widget_version', '2.10.2')
+widget_version = settings.UPLOADCARE.get('widget_version', '2.x')
```
Comment (homm) [inline pyuploadcare/dj/conf.py]: it's too radically :(

**Pull request merged**